pkg/network/dns/cache.go

```diff
@@ -201,8 +185,8 @@ func (c *reverseDNSCache) Expire(now time.Time) {
 	total := len(c.data)
 	c.mux.Unlock()
 
-	c.expired.Store(int64(expired))
-	c.length.Store(int64(total))
+	cacheTelemetry.expired.Set(int64(expired))
```

Comment:
Why is this a gauge? It seems to be measuring the number of entries expired in this call. I think a counter makes more sense, WDYT?

Reply:
yes agreed will switch